client/connector/create.go

```diff
@@ -60,7 +60,7 @@ func Create(ctx context.Context, client http.Client, createInp CreateInput) (*Cr
 		untilCommunicationQueueReady(ctx, client, *NewReadByUidInput(createOutp.Uid)),
 		retry.NewOptionsBuilder().
 			Message("Waiting for connector to be created....").
-			Retries(10).
+			Retries(-1).
```

Comment:
Should we change this value to 20 and double it? 

Inifitely retrying seems like it would lead to hanging for customers. Causing issues with their terraform locks